docs/_snippets/page-story-slots.md

```diff
@@ -171,24 +171,23 @@ export const CustomFooter: Story = {
 };
 ```
 
-```html renderer="svelte" language="ts" tabTitle="native-format"
-{/* Page.stories.svelte */}
-
-<script>
-  import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
-
-  import Page from './Page.svelte';
-</script>
-
-<meta title="Page" component="{Page}" />
+```svelte filename="Page.stories.svelte" renderer="svelte" language="js" tabTitle="Svelte CSF"
```

Comment:
this is doable in Svelte CSF, but skipping this for now due to time constraints.